_posts/2015-09-16-introduction-of-recommender-algorithms-for-e-commerce.md

```diff
@@ -0,0 +1,65 @@
+
+Hence, it is not surprising that Recommender Systems are essential part of e-commerce solutions in this era. At present day, many recommender algorithms exist which can be employed into recommender systems built for e-commerce. In this blog-post we will have high level overview of the classification of such recommender algorithms.
+
+
+##Algorithms
+
+Recommender Algorithms are mainly classified into two categories as described below:
+
+1. ###Content based recommendation systems
+> Content-based recommendation systems recommend an item to a user based upon a description of the item and a profile of the user’s interests. While a user profile may be entered by the user, it is commonly learned from feedback and/or the interaction of the user on items. A variety of learning algorithms have been adapted to learning user profiles. [1]
+>
+> Algorithms: Relevance feedback, genetic algorithms, neural networks, and the Bayesian classifier are among the learning techniques for learning a user profile. The vector space representation and latent semantic indexing can both be used by these learning methods to represent documents.
+>
+> **Pros:** It does not rely on other user's feedback. Provides purely feature based recommendations. It Does not suffer from cold start.
+>
+> **Cons:** Limited content analysis and limited degree of novelty. Lower recommendation value.
+>
+2. ###collaborative-filtering
+> From all different approaches for obtaining recommendations, the most popular one is by far the collaborative-filtering. This approach makes recommendations by discovering correlation between users of the recommender system. It is a unified approach for discovering the list of potentially interesting items (items not been accessed by the active user) and predicting its relevancy to the active user. The idea is to reach the final recommendation only on the basis of similar users and their actions. [6]
+>
+> Two main categories of collaborative-filtering algorithms:
+>
+> 1. **User-based Collaborative Filtering (Memory-based):** Memory-based algorithms utilize the entire user-item database to generate a prediction. These systems employ statistical techniques to find a set of users, known as neighbors, that have a history of agreeing with the target user (i.e., they either rate different items similarly or they tend to buy similar set of items). Once a neighborhood of users is formed, these systems use different algorithms to combine the preferences of neighbors to produce a prediction or top-N recommendation for the active user. The techniques, also known as nearest-neighbor or user-based collaborative filtering are more popular and widely used in practice. User-based collaborative filtering systems have been very successful in past, but their widespread use has revealed some potential challenges such as Sparsity and Scalability.[2]
```

Comment:
Last sentence: Is it necessary to start "sparsity" and "scalability" with an uppercase character?